pygmt/tests/test_ternary.py

```diff
@@ -0,0 +1,23 @@
+"""
+Tests ternary.
+"""
+import pytest
+from pygmt import Figure
+
+
+@pytest.mark.mpl_image_compare
+def test_ternary():
+    """
+    Test plotting a ternary chart.
+    """
+    fig = Figure()
+    fig.ternary(
+        data="@ternary.txt",
```

Comment:
Need to cache this file using the new function `load_sample_data`.

Reply:
Does this need to be cached for `load_sample_data` or is it enough to cache it for the tests?

Reply:
I think it depends on if you want to add an example for ternary. If the cache data is used in an example, I think it would be better to add it to the `load_sample_data` function. Of course, it can/should be done in a separate PR. So, caching it in CI is enough for this PR.

Reply:
Just one more thought on the test: perhaps we should use a much simpler dataset (with only a few records) in a test, so that it's easier to verify if the test is correct or not. The current test can be a good example instead.

Reply:
I changed this to use a simple dataset